packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -224,6 +225,34 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	@Input()
 	public rowClassNameFn?: (row: object, index: number) => string;
 
+	/**
+	 * Determine if the row index must be present or not.
+	 * Default: true
```

Comment:
Shouldn't this be false?

Reply:
In fact, what's wrong in the source code... it should be initialized as `true` :p

Reply:
Previously, it was `true` by default. Do we keep the same behavior so ? 😊

Reply:
Yep, I think we should keep the same behavior as in Stark 9...

Reply:
To me it makes more sense that it's `false` by default.
And it also seems to work like this at the moment

Reply:
Currently, it's false by default (I didn't want to force everyone to have an index by default).
But, in the previous implementation of Stark (v9), it was enabled by default... 
I don't feel comfortable to enable this feature by default actually 😕

Reply:
Ok, so let's keep it as disabled by default in Stark 10 😉